### DISCUSSION TITLE
Boxed Layout - Sidebar next to content

### DISCUSSION THREAD

**Discussion by huteck** [templaza/astroid-framework#257] [Q&A]
Hi,

I would like to use a boxed layout with a sidebar menu BUT if I select sidebar that one is shown on the total left of the screen, then there is the background space till the boxed Layout.

Is there any option that the Sidebar is directly connected to the boxed side? -> "Background" - "Sidebar" - "Boxed content" - "Background"

![image](https://user-images.githubusercontent.com/117048767/205765297-0ebcb266-e81a-47f5-a32b-cf5eb5d0ea05.png)

**Comment by Chacapamac** [comment]
**Yes you right, this is the way the Sidebar with boxed layout work**

![Screen Shot 2022-12-20 at 11 09 58 AM](https://user-images.githubusercontent.com/1907276/208719114-a4039554-4d3b-45d1-95e2-edf568af8c82.jpg)

**If I understand, you like to have the sidebar be at the left of the content.**
The way to do so, I guess will be to have the **astroid-header** and **astroid-offcanvas** be inside the **astroid-wrapper** — To me that appear to be a big customization?

Maybe @sonvnn will have a better idea?
I will try again....
